apps/rpc/rpc_server.cc

```diff
@@ -0,0 +1,243 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file rpc_server.cc
+ * \brief RPC Server for TVM.
+ */
+#include <stdlib.h>
+#include <iostream>
+#include <signal.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <cstring>
+#include <vector>
+#include <sstream>
+
+#include "../../src/runtime/rpc/rpc_base.h"
+#include "../../src/runtime/rpc/rpc_server.h"
+
+using namespace tvm::runtime;
+
+#define USAGE \
```

Comment:
It would be more safe to use `static const std::string kUSAGE = "...";` or `static const char * kUSAGE = "";`.